styles/lichess/catppuccin.user.css

```diff
@@ -1,5 +1,5 @@
 /* ==UserStyle==
-@name Lichess Catppuccin
+@name lichess Catppuccin
```

Comment:
I think we should make the name uppercase in `userstyles.yml` (and here), see https://github.com/catppuccin/userstyles/pull/346#discussion_r1398709738 for the conversation from a while back